### DISCUSSION TITLE
Cannot delete Classification Store from GUI

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10070] [Q&A / Support] [answered]
We can delete Classes, Field-Collections, Objectbricks... with right mouse click but not for Classification Store.
Why?

**Comment by weisswurstkanone** [comment, marked as answer]
No reason, just a missing feature 😄 Could you provide a PR please ?

**Reply by ghost** [reply]
Not strong enough for a Pull Request...
